BankingKataSpecs/AccountSpecs/Features/DepositCash.feature

```diff
@@ -0,0 +1,10 @@
+﻿Feature: DepositCash
+	In order to deposit cash into my account
+	As a foo user
+	I want to be able to deposit Cash
+
+@mytag
+Scenario: Depositing my cash
+	Given I create an account with 0 cash
```

Comment:
You can bring up a list of existing steps to pick from but I haven't found any refactoring support for them.